installing/installing_alibaba/installation-config-parameters-alibaba.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY
+[id="installation-config-parameters-alibaba"]
+= Installation configuration parameters for Alibaba Cloud
+include::_attributes/common-attributes.adoc[]
+:context: installation-config-parameters-alibaba
+:platform: Alibaba Cloud
+
+toc::[]
+
+Before you deploy an {product-title} cluster on Alibaba Cloud, you provide parameters to customize your cluster and the platform that hosts it. When you create the `install-config.yaml` file, you provide values for the required parameters through the command line. You can then modify the `install-config.yaml` file to customize your cluster further.
```

Comment:
This is just a suggestion, but I almost feel like there should be a word after you and provide in this paragraph. The [style guide](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#second-person) has something similar listed in one of the examples. 

```suggestion
Before you deploy an {product-title} cluster on Alibaba Cloud, you must provide parameters to customize your cluster and the platform that hosts it. When you create the `install-config.yaml` file, you must provide values for the required parameters through the command line. You can then modify the `install-config.yaml` file to customize your cluster further.
```

Reply:
I'd say if not `must` maybe `can`.